roles/core/cluster/tasks/cluster.yml

```diff
@@ -74,6 +74,35 @@
     - scale_cluster_profile_name is defined and scale_cluster_profile_name != 'None'
     - scale_cluster_profile_name not in gpfs_cluster_system_profile
 
+- block:
+    - name: check | cluster profile name validation
+      assert:
+        that:
+          - scale_cluster_profile_name is not match("gpfs.*")
+        msg: >-
+          A user-defined profile must not begin with the string 'gpfs'
+
+    - name: check | cluster profile format validation
+      assert:
+        that:
+          - stat_user_profile_result.matched == 1
+        msg: >-
+          A user-defined profile must have the .profile suffix
+  delegate_to: localhost
+  when:
+    - scale_cluster_profile_name is defined and scale_cluster_profile_name != 'None'
+    - scale_cluster_profile_name not in gpfs_cluster_system_profile
+
+- block:
+    - name: cluster | Copy user defined profile
+      copy:
+        src: "{{ stat_user_profile_result.files.0.path }}"
+        dest: "{{ scale_cluster_profile_system_path }}"
+        mode: a+x
+  when:
+    - scale_cluster_profile_name is defined and scale_cluster_profile_name != 'None'
+    - scale_cluster_profile_name not in gpfs_cluster_system_profile
```

Comment:
Why is a block needed here on a single task? Seems like the above has the same exact conditions, except that it's run on localhost, and this one is not?